### PR TITLE
132 tests for the dao

### LINKED ISSUE #132: Write tests for the dao
Right now dao module is not tested at all, we need to test it to improve the quality of the software

### PR DESCRIPTION
**Pull request by BritikovKI**
Closes #132

### CODE REVIEW

File: packages/dao/contracts/Api3BaseTemplate.sol [hunk 1]
```diff
@@ -19,6 +19,8 @@ import "@aragon/os/contracts/factory/DAOFactory.sol";
 import "@aragon/os/contracts/common/IsContract.sol";
 import "@aragon/os/contracts/common/Uint256Helpers.sol";
 import "@aragon/id/contracts/IFIFSResolvingRegistrar.sol";
+//TODO: This is added to use this contract in tests, I think this is stupid, need to think ho to change it
```
Comment (kinspotting) [inline packages/dao/contracts/Api3BaseTemplate.sol:22]: we can add `test/Imports.sol` and import it there

**Review by kinspotting — CHANGES_REQUESTED** [review]
see comments

File: packages/dao/contracts/Api3BaseTemplate.sol (hunk 1 above)
Comment (BritikovKI) [reply packages/dao/contracts/Api3BaseTemplate.sol:22]: To be honest, that's kind of crutch as well, but it is definitely a better soution...

File: packages/dao/contracts/test/Imports.sol [hunk 2]
```diff
@@ -0,0 +1,8 @@
+pragma solidity 0.4.24;
+
+
+//TODO: This is added to use this contract in tests, I think this is stupid, need to think ho to change it
```
Comment (kinspotting) [inline packages/dao/contracts/test/Imports.sol:4]: delete this comment pls :)

**Review by kinspotting — COMMENTED** [review]
small changes still needed

File: packages/dao/contracts/test/Imports.sol (hunk 2 above)
Comment (BritikovKI) [reply packages/dao/contracts/test/Imports.sol]: Deleted, I still think that's not an ideal solution though 😂

**Pull request merged**